### DISCUSSION TITLE
Feature request: Click to zoom on heading or task

### DISCUSSION THREAD

**Discussion by jamie9090** [vslinko/obsidian-zoom#102] [Ideas]
Could there be a visual way to click to zoom into a heading or task, similar to how click to zoom on bullet works?

**Comment by tobiasbuckell** [comment]
Click to zoom on heading would be amazing!

**Reply by ACGarland** [reply]
Another vote for zooming on heading... for several reasons:
- The core outline view doesn't recognized bullets, only headings.  So there is presently a bit of a functionality mismatch: if I want to use zoom, I have to use bullets (not visible in core outline view).  If I want to use the core outliner view, I have to use headings (no zoom).
- Heading show up in the zoomed navigation context (e.g., `heading1` > `heading2` > `bulletA` > `bulletA1`) and you can click on them there to zoom out... but not within the document to zoom in. This is inconsistent.

**Comment by PlasticFeather** [comment]
I really want this feature
